napari/utils/settings/_defaults.py

```diff
@@ -0,0 +1,123 @@
+"""Settings management.
+
+theme (superseding #943)
+PySide2 vs PyQt5 preference (if both installed)
+window position/geometry
+opt in for telemetry
+font size for console
+
+monitor DPI (#820)
+call order of plugins (after #937)
+default colormaps or color combinations (as discussed in #619)
+magic-naming for layer (off by default in #1008)
+highlight thickness for shapes / points layers
+dask cache size maximum, and dask fusion settings (#1173 (comment))
+version updates (has user been asked)
+key/mouse bindings
+"""
+
+from enum import Enum
+from typing import List, Tuple
+
+from pydantic import BaseSettings, Field
+
+from napari.utils.events.evented_model import EventedModel
+
+# If a plugin registers a theme, how does it work to do model generation?
+# FIXME: Generating enums on the fly for Themes?
+
+
+class QtBindingEnum(str, Enum):
+    """Python Qt binding to use with the application."""
+
+    pyside = 'pyside2'
+    pyqt = 'pyqt5'
+
+
+class ThemeEnum(str, Enum):
+    """Python Qt binding to use with the application."""
+
+    dark = 'dark'
+    light = 'light'
+
+
+class ApplicationSettings(BaseSettings, EventedModel):
+    """Main application settings."""
+
+    version = (0, 1, 0)
+    # Python
+    qt_binding: QtBindingEnum = QtBindingEnum.pyside
+    # UI Elements
+    highlight_thickness: int = 1
+    theme: ThemeEnum = ThemeEnum.dark
+    # Startup
+    opt_in_telemetry: bool = Field(
+        False, description="Check to enable telemetry measurements"
+    )
```

Comment:
thoughts:

Make that a date time; so if telemetry ever changes, we can know wether user have accepted before/after the change , and/or maybe remind them that we are collecting every year or so ?

Reply:
Good one! thanks!